package.json

```diff
@@ -82,6 +82,15 @@
     "semi": true,
     "arrowParens": "avoid"
   },
+  "tsd": {
+    "directory": "type-definitions/ts-tests",
+    "compilerOptions": {
+      "baseUrl": "type-definitions/ts-tests",
+      "paths": {
+        "immutable": ["../../type-definitions/immutable.d.ts"]
+      }
```

Comment:
Seems to be specified in `./type-definitions/ts-tests/tsconfig.json`: https://github.com/immutable-js/immutable-js/blob/ac05955988f47da9f6fb97279a36de3e7ffbb700/type-definitions/ts-tests/tsconfig.json#L13-L15

Reply:
Indeed it is. I was unable to make tsd read that `tsconfig.json` file but did not put much time into it. If you know how, please share.

Reply:
Hard to say. Might be worth opening an issue in their repo.

Some time ago I was maintaining a forked `tsd-lite`. All I remember is that the original config logic was rather some spaghetti code. It was fully replaced in the fork.